src/features/staking/utils/convertBigIntToBigNumber.ts

```diff
@@ -0,0 +1,9 @@
+import BigNumber from "bignumber.js";
+
+export const convertBigIntToBigNumber = (balance: bigint | number) => {
```

Comment:
This function is badly named - it's returning a `number` and not `bigint` or `BigNumber`.
I think I would call it something like `bigIntToUnits`, and not accept a `number` as a param.